CondFormats/SiPixelTransient/src/SiPixelTemplate2D.cc

```diff
@@ -166,10 +166,9 @@ bool SiPixelTemplate2D::pushfile(int filenum, std::vector<SiPixelTemplateStore2D
 
     // next, layout the 2-d structure needed to store template
 
-    theCurrentTemp.entry = new SiPixelTemplateEntry2D*[theCurrentTemp.head.NTyx];
-    theCurrentTemp.entry[0] = new SiPixelTemplateEntry2D[theCurrentTemp.head.NTyx * theCurrentTemp.head.NTxx];
-    for (int i = 1; i < theCurrentTemp.head.NTyx; ++i)
-      theCurrentTemp.entry[i] = theCurrentTemp.entry[i - 1] + theCurrentTemp.head.NTxx;
+    theCurrentTemp.entry.resize(theCurrentTemp.head.NTyx);
+    for (int iy = 0; iy < theCurrentTemp.head.NTyx; ++iy)
+      theCurrentTemp.entry[iy].resize(theCurrentTemp.head.NTxx);

```

Comment:
```suggestion
    for (auto& item : theCurrentTemp.entry)
      item.resize(theCurrentTemp.head.NTxx);
```
is marginally shorter to write and may even be epsilon faster